### PR TITLE
cypress initial setup

### PR DESCRIPTION
**Pull request by dtthuynh**
Initial PR for Cypress Tests.

This initial commit includes one sample test where we just login to ACM.

### CODE REVIEW

**Review by chrisahl — COMMENTED** [review]
So all the files under test/e2e/cypress-ui/cypress/sample are for a simple test or utilities to be used by Cypress?  Just seems like a lot of files so wanted to be suer this wasn't for a generic cypress example...

**Comment by dtthuynh** [comment]
> So all the files under test/e2e/cypress-ui/cypress/sample are for a simple test or utilities to be used by Cypress? Just seems like a lot of files so wanted to be suer this wasn't for a generic cypress example...

Tests will be under cypress/tests - you are correct that the ones under /sample are generic cypress examples. I'll remove them.

File: test/e2e/cypress-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# IDP Cypress Tests
+
+## Running the tests locally
+
+  > Pre-requisites:
+  >  - nodeJS
+  >  - OpenShift CLI (oc)
+  >  - a browser - either chrome or firefox
+
+1. Run `npm install` to install Cypress and it's dependencies on the root directory.
```
Comment (vidyanambiar) [inline test/e2e/cypress-ui/README.md:10]: The `package.json` file is not present in the root directory. I believe this line would need to indicate that `npm install` needs to be run from the `test/e2e/cypress-ui ` directory.

**Review by vidyanambiar — COMMENTED** [review]
Thanks for adding this, David!

I ran the acmLogin test using `npx cypress run` and the openshift cluster login page came up as expected, but the kubeabdmin login did not go through. Is that expected? I wasn't sure if that was meant to be part of the spec or not.

**Pull request merged**